lib/matplotlib/patches.py

```diff
@@ -1037,13 +1037,24 @@ def _update_path(self):
                              f"`len(values) = {self._values.size}` and "
                              f"`len(edges) = {self._edges.size}`.")
         verts, codes = [], []
-        for idx0, idx1 in cbook.contiguous_regions(~np.isnan(self._values)):
+
+        _nan_vals = ~np.isnan(self._values)
```

Comment:
The naming is a bit misleading, maybe one of `not_nan_vals`, `finite_vals`, `valid_vals`.